pkg/controllers/vdb/drainnode_reconciler.go

```diff
@@ -91,12 +92,15 @@ func (s *DrainNodeReconciler) Reconcile(ctx context.Context, _ *ctrl.Request) (c
 	elapsed := time.Since(drainStartTime)
 	timeout := time.Duration(timeoutInt) * time.Second
 	// If timeout has expired, we move on to the next reconciler (DBRemoveNodeReconciler|DBRemoveSubclusterReconciler)
+	s.VRec.Log.Info("Draining in progress", "start", drainStartTime, "elapsed", elapsed, "timeout", timeout)
 	if elapsed >= timeout {
 		s.VRec.Log.Info("Draining timeout has expired")
+		s.killConnectionsToPendingDeletePods(ctx, pfs)
```

Comment:
I added killing connections back. Because in remove_node/subcluster reconcilers, there is a very small window where if vdb has changed since the last collection, we will requeue and go through draining again. But killing sessions is just best-effort, even if it fails, we still proceed with remove_node/subcluster as they will kill the sessions anyway